app.js

```diff
@@ -24,6 +25,12 @@ mongoose.connect(app.get('mongourl'), function(err) {
     if (err) throw err;
 });
 
+raven
+    .config(process.env.RAVEN_DSN)
```

Comment:
Don't think we need the production check here, see https://docs.sentry.io/clients/node/usage/#disable-raven